.github/assets/test_scripts/sftp_profile_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# This script is used to test the SFTP profile functionality of the OCI CLI.
+# It is intended to be run either from the OCI CLI's test suite or by itself, run from the base oci_env directory.
+
+ENV_FILE=".github/assets/test_scripts/sftp_compose.env"
+
+oci-env -e $ENV_FILE compose build
+oci-env -e $ENV_FILE compose up -d
+
+oci-env -e $ENV_FILE poll
+
+oci-env -e $ENV_FILE pulp file content upload --file "dev_requirements.txt" --relative-path "dev_requirements.txt"
+
```

Comment:
```suggestion

pulp file content upload --file "dev_requirements.txt" --relative-path "dev_requirements.txt"

```

Reply:
The oci env `pulp` command should actually run the pulp inside the built container, and therefore using the proper instance to test with. Now it is even more ambiguous where that dev_requirement.txt file should come from.

Reply:
+1 to what Matthias said, about the dev_requirements, it is a file which gets generated in the oci-env container everytime, so it saves the effort of also connecting to the container, creating some test_file, etc.